samples/OneToOneCall/webpack.config.js

```diff
@@ -1,9 +0,0 @@
-// Copyright (c) Microsoft Corporation.
```

Comment:
@prprabhu-ms I deleted this because it seemed like it was forgotten in the OneToOne deletion PR?